extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteUnionQueryMSQTest.java

```diff
@@ -62,13 +62,12 @@ protected QueryTestBuilder testBuilder()
    */
   @Test
   @Override
-  public void testUnionIsUnplannable()
+  public void testUnionIsUnplannableInNative()
```

Comment:
Aside from the name change, why does this evaluate native-only?

Reply:
the original wording have made me end up with an unfortunate name; renamed it to `testUnionDifferentColumnOrder`